src/components/output/custom-els/MultiPanel/styles.css

```diff
@@ -0,0 +1,11 @@
+multi-panel > .panel-heading {
```

Comment:
Think we could go with sass here? I believe the other web components are using it already.

Reply:
hmmm,  I simply followed PinchZoom and TwoUp which are css.